### PR TITLE
EPMRPP-39153. Redesign pattern tab.

### PR DESCRIPTION
**Pull request by ElenaRomanchuk**
Added PA and AA icons on step and log view (latest changes). 
Created rename modal instead of using create modal on edit pattern.
Sticky header in pattern analysis tab.

### CODE REVIEW

File: app/src/pages/common/settingsPage/patternAnalysisTab/patternList/patternListItem/patternControlPanel/patternControlPanel.jsx [hunk 1]
```diff
@@ -61,15 +57,13 @@ export class PatternControlPanel extends Component {
     readOnly: false,
   };
 
-  onEditPattern = () => {
-    const { intl, showModal, updatePattern, pattern } = this.props;
+  onRenamePattern = () => {
```
Comment (AmsterGet) [inline app/src/pages/common/settingsPage/patternAnalysisTab/patternList/patternListItem/patternControlPanel/patternControlPanel.jsx]: Seems that you forget to rename function [here](https://github.com/reportportal/service-ui/pull/1597/files#diff-8da86751ed05967963f336b5325068a7R134)

File: app/src/pages/common/settingsPage/patternAnalysisTab/patternListHeader/patternListHeader.scss [hunk 2]
```diff
@@ -1,12 +1,18 @@
 .pattern-list-header {
+  position: sticky;
+  top: 0px;
```
Comment (AmsterGet) [inline app/src/pages/common/settingsPage/patternAnalysisTab/patternListHeader/patternListHeader.scss]: Just `top: 0`

File: app/src/pages/inside/stepPage/stepGrid/defectType/autoAnalyzedLabel/autoAnalyzedLabel.jsx [hunk 3]
```diff
@@ -1,6 +1,8 @@
+import Parser from 'html-react-parser';
 import classNames from 'classnames/bind';
+import AAIcon from '../img/aa_activated-inline.svg';
```
Comment (AmsterGet) [inline app/src/pages/inside/stepPage/stepGrid/defectType/autoAnalyzedLabel/autoAnalyzedLabel.jsx]: Please move these icons to the `common/img`

Comment (ElenaRomanchuk) [reply app/src/pages/inside/stepPage/stepGrid/defectType/autoAnalyzedLabel/autoAnalyzedLabel.jsx]: Fixed.

File: app/src/pages/common/settingsPage/patternAnalysisTab/patternListHeader/patternListHeader.scss (hunk 2 above)
Comment (ElenaRomanchuk) [reply app/src/pages/common/settingsPage/patternAnalysisTab/patternListHeader/patternListHeader.scss]: Fixed.

File: app/src/pages/common/settingsPage/patternAnalysisTab/patternList/patternListItem/patternControlPanel/patternControlPanel.jsx (hunk 1 above)
Comment (ElenaRomanchuk) [reply app/src/pages/common/settingsPage/patternAnalysisTab/patternList/patternListItem/patternControlPanel/patternControlPanel.jsx]: Fixed.

**Pull request merged**